tutorcairn/templates/cairn/apps/superset/superset_config.py

```diff
@@ -162,4 +162,13 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
     "EMBEDDED_SUPERSET": True
 }
 
+# Embedded Dashboard CORS
+ENABLE_CORS=True
+CORS_OPTIONS={
+    "supports_credentials": True,
+    "allow_headers": ["*"],
+    "resources": ["*"],
```

Comment:
can you comment on why are all these configs needed? The origins list is obvious, but the others need a bit more context.